src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1779,7 +1801,8 @@ private void visitTypeDeclaration(JSType type, boolean isVarArgs, boolean isOpti
      * the literal string that was in the original code. Special care is taken for templatized
      * types.
      */
-    private void emitNoResolvedTypeOrDefault(NoType type, String defaultEmit) {
+    private void emitNoResolvedTypeOrDefault(
```

Comment:
Add some documentation for `emitInstanceForObject`.

Reply:
Done.